requests_openapi_client/core.py

```diff
@@ -220,7 +220,9 @@ def f(client, **kwargs):
                     pass
                 raise e
 
-            data = response.json()
+            data = (
+                response.json() if int(response.headers["Content-Length"]) > 0 else None
```

Comment:
@bvacherot-sofar I wonder if maybe it would be better to just do `len(response.content)` instead? `Content-Length` is an optional header, and this potentially introduces a new crash if it's not present.